Pipfile

```diff
@@ -0,0 +1,40 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "==2.3.1"
+flake8 = "==3.7.8"
+black = "==19.3b0"
+requests = "==2.22.0"
+python-dotenv = "==0.10.3"
+Flask = "==1.1.1"
+flask-restful = "==0.3.7"
+flask-httpauth = "==3.3.0"
+flask-sqlalchemy = "==2.4.1"
+flask-marshmallow = "==0.10.1"
+marshmallow = "==2.19.5"
+marshmallow-sqlalchemy = "==0.19.0"
+psycopg2-binary = "==2.8.4"
+loguru = "==0.4.1"
+
+[packages]
+pylint = "==2.3.1"
+flake8 = "==3.7.8"
+black = "==19.3b0"
```

Comment:
We can just put these package in [dev-packages]

Reply:
Fix in b36cae8.